mypy.ini

```
@@ -0,0 +1,2 @@
+[mypy]
+ignore_missing_imports = True
```

nit: trailing newline

```suggestion
ignore_missing_imports = True

```